ex2_hal/athena/equipment_handler/source/board_io_tests.c

```diff
@@ -37,8 +37,8 @@ void InitIO(void) {
 #else
     gioSetDirection(hetPORT1, 0x9CFF7BEF);
 #endif
-    gioSetDirection(gioPORTA, 0xFFFFFF6F);
-    gioSetDirection(gioPORTB, 0xFFFFFFFF);
+    //gioSetDirection(gioPORTA, 0xFFFFFF6F);
```

Comment:
Are these magic numbers documented in the code anywhere?

Reply:
They correspond to the directions of each pin. There is a task to remove them and instead set the hardware configuration